RadarSDK/RadarVerificationManager.m

```diff
@@ -328,4 +337,230 @@ - (void)getAttestationWithNonce:(NSString *)nonce completionHandler:(RadarVerifi
     }
 }
 
+// inspired by https://github.com/securing/IOSSecuritySuite
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wundeclared-selector"
+- (BOOL)isJailbroken {
+    NSError *error = nil;
+    NSFileManager *fileManager = [NSFileManager defaultManager];
+    
+    // URL scheme check
+    NSArray *suspiciousURLSchemes = @[
+        @"undecimus://",
+        @"sileo://",
+        @"zbra://",
+        @"filza://"
+    ];
+    for (NSString *urlScheme in suspiciousURLSchemes) {
+        NSURL *url = [NSURL URLWithString:urlScheme];
+        if ([[UIApplication sharedApplication] canOpenURL:url]) {
+            [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelDebug message:@"Jailbreak check failed: URL scheme check"];
+            return YES;
```

Comment:
Perhaps right now, instead of early returning `YES` we store the bool and return at the end so we get debug logs for all the individual checks that are failing. Probably good data points to have.

Reply:
That's a good call, will update

Reply:
Done